### PR TITLE
feat(query): Added SLB Policy With Insecure TLS Version In Use for Terraform Alicloud

### PR DESCRIPTION
**Pull request by cxMiguelSilva**
**Proposed Changes**
- SLB Policy With Insecure Tls Version In Use

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/alicloud/slb_policy_with_insecure_tls_version_in_use/metadata.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "dbfc834a-56e5-4750-b5da-73fda8e73f70",
+  "queryName": "SLB Policy With Insecure Tls Version In Use",
+  "severity": "MEDIUM",
+  "category": "Encryption",
+  "descriptionText": "SLB Policy should not support insecure versions of Tls protocol",
```
Comment (rafaela-soares) [inline assets/queries/terraform/alicloud/slb_policy_with_insecure_tls_version_in_use/metadata.json:6]:
```suggestion
  "queryName": "SLB Policy With Insecure TLS Version In Use",
  "severity": "MEDIUM",
  "category": "Encryption",
  "descriptionText": "SLB Policy should not support insecure versions of TLS protocol",
```

for standard purposes

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/terraform/alicloud/slb_policy_with_insecure_tls_version_in_use/query.rego [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+CxPolicy[result] {
+	resource := input.document[i].resource.alicloud_slb_tls_cipher_policy[name]
+	tls_version := insecureVersions[_]
+	resource.tls_versions[_] == tls_version
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("alicloud_slb_tls_cipher_policy[%s].tls_versions", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": sprintf("alicloud_slb_tls_cipher_policy[%s].tls_versions uses secure TLS versions", [name]),
```
Comment (joaorufi) [inline assets/queries/terraform/alicloud/slb_policy_with_insecure_tls_version_in_use/query.rego:16]:
```suggestion
		"keyExpectedValue": sprintf("alicloud_slb_tls_cipher_policy[%s].tls_versions to use secure TLS versions", [name]),
```

File: assets/queries/terraform/alicloud/slb_policy_with_insecure_tls_version_in_use/test/positive_expected_result.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "queryName": "SLB Policy With Insecure TLS Version In Use",
+    "severity": "MEDIUM",
+    "line": 3
+  }
+]
```
Comment (joaorufi) [inline assets/queries/terraform/alicloud/slb_policy_with_insecure_tls_version_in_use/test/positive_expected_result.json:7]: Could you add `fileName` ?

**Review by joaorufi — APPROVED** [review]

File: assets/queries/terraform/alicloud/slb_policy_with_insecure_tls_version_in_use/query.rego (hunk 2 above)
Comment (joaorufi) [inline assets/queries/terraform/alicloud/slb_policy_with_insecure_tls_version_in_use/query.rego:16]:
```suggestion
		"keyExpectedValue": sprintf("alicloud_slb_tls_cipher_policy[%s].tls_versions to use secure TLS versions", [name]),
```

File: assets/queries/terraform/alicloud/slb_policy_with_insecure_tls_version_in_use/test/positive_expected_result.json (hunk 3 above)
Comment (joaorufi) [inline assets/queries/terraform/alicloud/slb_policy_with_insecure_tls_version_in_use/test/positive_expected_result.json:7]: Could you add `fileName` ?

**Review by joaorufi — CHANGES_REQUESTED** [review]

**Review by joaorufi — APPROVED** [review]

**Review by rafaela-soares — APPROVED** [review]

**Pull request merged**